src/cache/RedisQueryResultCache.ts

```diff
@@ -73,22 +73,18 @@ export class RedisQueryResultCache implements QueryResultCache {
      */
     getFromCache(options: QueryResultCacheOptions, queryRunner?: QueryRunner): Promise<QueryResultCacheOptions|undefined> {
         return new Promise((ok, fail) => {
-
-            if (options.identifier) {
-                this.client.get(options.identifier, (err: any, result: any) => {
-                    if (err) return fail(err);
-                    ok(JSON.parse(result));
-                });
-
-            } else if (options.query) {
-                this.client.get(options.query, (err: any, result: any) => {
-                    if (err) return fail(err);
-                    ok(JSON.parse(result));
-                });
-
-            } else {
+            let key = options.identifier || options.query;
+            if (!key) {
                 ok(undefined);
             }
+            if (options.prefix) {
+                key = `${options.prefix}~${key}`;
```

Comment:
what is `~` here?

Reply:
Is simply a delimiter that would help it be more human readable when looking directly in the cache store. I can remove it if you would like?

Reply:
probably yes. let them decide and put this symbol in prefix if needed

Reply:
Sure no problem